bob-apps/bin/bobp.rs

```diff
@@ -45,11 +47,18 @@ impl NetConfig {
         Self {
             port: matches.value_or_default("port"),
             target: matches.value_or_default("host"),
+            ca_cert_path: matches.value_of("ca_path").map(|p| p.to_string()),
         }
     }
 
     async fn build_client(&self) -> BobApiClient<Channel> {
-        let endpoint = Endpoint::from(self.get_uri()).tcp_nodelay(true);
+        let mut endpoint = Endpoint::from(self.get_uri()).tcp_nodelay(true);
```

Comment:
`get_uri` uses `http` scheme. Have you checked that it works properly when TLS enabled?